### DISCUSSION TITLE
Use GBM to prune features, but do not use GBM as base model when performing ensemble

### DISCUSSION THREAD

**Discussion by jmren168** [autogluon/autogluon#3883] [General]
We're trying to train our customized models, and then use GBM to prune features. However, after feature pruning, GBM will be one of base models in the stage of ensemble. Is it possible to not use GBM when performing ensemble?

Settings:
`        # use gbm for feature_prune
        custom_model_hyperparameters.update({
            'GBM':{}
        })
        # Define feature_prune_kwargs
        feature_prune_kwargs = {
            'n_train_subsample': 50000,
            'n_fi_subsample': 10000,
            'prune_ratio': 0.05,
            'stopping_round': 10,
            'min_improvement': 1e-6,
            'force_prune': True,
        }`

Results:
![image](https://github.com/autogluon/autogluon/assets/31725302/b1127e16-f496-4132-91d0-f87711bd5647)

Expcected Results:
Only **CustomPipelineRFModel** are used as base models in WeightedEnsmeble.

**Comment by Innixma** [comment]
I created a GitHub issue to track adding support for this: #3890
